euphoria-hadoop/build.gradle

```diff
@@ -1,7 +1,20 @@
 dependencies {
   compile project(':euphoria-core')
-  compileOnly "org.apache.hadoop:hadoop-common:${hadoopVersion}"
-  compileOnly "org.apache.hadoop:hadoop-mapreduce-client-core:${hadoopVersion}"
+  compile "org.slf4j:slf4j-api:${slf4jVersion}"
+  compile "org.slf4j:log4j-over-slf4j:${slf4jVersion}"
+  compileOnly("org.apache.hadoop:hadoop-common:${hadoopVersion}") {
+    exclude group: 'log4j'
+    exclude group: 'org.slf4j'
+  }
+  compileOnly("org.apache.hadoop:hadoop-mapreduce-client-core:${hadoopVersion}") {
+    exclude group: 'log4j'
+    exclude group: 'org.slf4j'
+  }
+  compileOnly("org.apache.hadoop:hadoop-yarn-common:${hadoopVersion}") {
+    exclude group: 'log4j'
+    exclude group: 'org.slf4j'
+  }
```

Comment:
I think you can exclude the dependency globally by something like
```
configurations {
  compile.exclude group: "log4j"
  compile.exclude group: "org.slf4j"
}
```

Reply:
wouldn't his also exclude slf4j that we provide?